cobra/cmd/project.go

```diff
@@ -130,6 +132,14 @@ func (p *Project) CmdPath() string {
 	return p.cmdPath
 }
 
+// findCmdSuffix returns the cmd dir start with file path separator.
+func findCmdSuffix(cmdDir string) string {
```

Comment:
This function seems not to find a suffix, but to just ensure that the string starts with `os.PathSeparator`. I think it should be renamed to reflect the functionality. Alternatively `p.absPath = strings.TrimSuffix(absPath, findCmdSuffix(findCmdDir(absPath)))` (see comment above) can be included in this function, so it actually does find and extract a suffix. If so, the line above would be: `p.absPath = findCmdSuffix(absPath)`.